examples/models/llama2/model.py

```diff
@@ -211,9 +211,9 @@ def __init__(self):
             **params,
         )
         self.model_ = Transformer(model_args)
-        self.model_.load_state_dict(
-            checkpoint, strict=False
-        )  # self.model_ = Transformer(gptconf)
+        # self.model_.load_state_dict(
```

Comment:
It is a hack for quickly test different llama size, but not for landing (because the actual checkpoint is missing).